ext/pdo_sqlite/sqlite_statement.c

```diff
@@ -434,6 +432,8 @@ static int pdo_sqlite_stmt_set_attribute(pdo_stmt_t *stmt, zend_long attr, zval
                     zend_value_error("explain mode must be one of the Pdo\\Sqlite::EXPLAIN_MODE_* constants");
                     return 0;
                 }
+
+                pdo_sqlite_stmt *S = (pdo_sqlite_stmt*)stmt->driver_data;
```

Comment:
The cast is also not necessary here.
```suggestion
                pdo_sqlite_stmt *S = stmt->driver_data;
```

Reply:
Technically not, but casting `void*` is somewhat of a personal preference. It's there for all other usages of `driver_data` in this file, so we should probably stay consistent.